clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -1058,8 +1148,10 @@
                                                                        new Field("partitions",
                                                                                  new ArrayOf(LEADER_AND_ISR_RESPONSE_PARTITION_V0)));
 
-    public static final Schema[] LEADER_AND_ISR_REQUEST = {LEADER_AND_ISR_REQUEST_V0};
-    public static final Schema[] LEADER_AND_ISR_RESPONSE = {LEADER_AND_ISR_RESPONSE_V0};
+    public static final Schema LEADER_AND_ISR_RESPONSE_V1 = LEADER_AND_ISR_RESPONSE_V0;
```

Comment:
Sure. I added this: `LeaderAndIsrResponse V1 may receive KAFKA_STORAGE_ERROR in the response`